### ISSUE TITLE
implement a toggle that turns the custom editor into a standard textarea

### ISSUE THREAD

**Issue by camlorn** [rust-lang/rust-playpen#198]
As a screen reader user, the playground doesn't work much if at all.  I can't input text, edit, or conveniently view code from others.  These boil down to the same problem: the custom editor used is not very accessible.

Making the custom component accessible also isn't really possible for a lot of stupid, complicated reasons.  But implementing a toggle that turns it into a standard multiline edit field without line numbers seems like it'd be an instant win for all of the issues I have with it.

It would also be useful if the playground remembered my setting.

I'd go do this, but I have a tendency to break CSS stuff without meaning to, and I obviously can't exactly test for that.  If someone is willing to work with me, I'm willing to try.

**Comment by jonas-schievink** [comment]
cc ajaxorg/ace#2164
